src/styles/hc-2018/sponsorship.styl

```diff
@@ -0,0 +1,56 @@
+#sponsorship-hack-cambridge-logo {
+    display: flex;
+}
+#sponsorship-hack-cambridge-logo-object {
+    width: 108.3242pt;
+    height: 93.8102pt;
+}
+#sponsorship-hack-cambridge-wordmark {
+    font-size: 21.3953315789pt;
+    margin-top: 28.6354569254pt;
+    margin-left: 13.1663578947pt;
+}
+#sponsorship-university-of-cambridge-logo-object {
```

Comment:
Were you intending to create a large space between the Cambridge logo and the related copy?

<img width="911" alt="Cambridge logo / copy space" src="https://user-images.githubusercontent.com/14129055/29796145-815dbf40-8c47-11e7-8829-f9c0a5e590d2.png">

Reply:
I have this (and just pulled in the changes in the branch to be sure):
<img width="482" alt="screen shot 2017-08-28 at 23 26 45" src="https://user-images.githubusercontent.com/18505817/29796328-6491ab6e-8c48-11e7-8e21-c5056ef373ca.png">

Reply:
I'm seeing the issue with the latest set of changes with Safari (but not Chrome)